### ISSUE TITLE
Files not given CodeScene score

### ISSUE THREAD

**Issue by jamieh-sky** [codescene-oss/codescene-vscode#72]
Hi,

It has been noticed that a few files are not given a CodeScene score even though other files of the same class type and in the same directory do. 
![2024-07-25_12h27_10](https://github.com/user-attachments/assets/d3884855-e503-4531-bd3c-0e1a55f88464)
![2024-07-25_12h27_21](https://github.com/user-attachments/assets/805f55f3-4247-4580-a000-0c6f68e62f65)
![2024-07-25_12h27_29](https://github.com/user-attachments/assets/1426e01c-12d0-42e2-bf3a-178755a086a4)
![2024-07-25_12h27_38](https://github.com/user-attachments/assets/913d8c9a-4db0-4134-8308-eb47ac605acd)

Is there any reason why some files might not be given a score?

VSCode:
Version: 1.91.1 (user setup)
Commit: f1e16e1e6214d7c44d078b1f0607b2388f29d729
Date: 2024-07-09T22:06:49.809Z
Electron: 29.4.0
ElectronBuildId: 9728852
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Windows_NT x64 10.0.22631

VSCode Ext version: v0.7.7

**Comment by adamtornhill** [comment]
Thanks for reaching out, @jamieh-sky. Code Health is a metric for application code. Could it be that the unscored files only contain declarations (e.g. event handlers)? That is, there aren't any methods in them?

**Comment by adamtornhill** [comment]
@jamieh-sky Sorry for the late reply here. We'll prioritize this issue. To do so, we would need a bit more context. Would it be possible to share a little bit of the code with us via a secure and private channel? Please reach out via email: help (at) codescene.com

**Comment by jlindbergh** [comment]
Hi @jamieh-sky !
In order to avoid reviewing minified files (like javascript bundles) we have a maximum line length limitation. Any file containing excessively long lines will be ignored by the reviewer. There should be a warning message in the CodeScene Log about that, but we'll improve the UX for that in an upcoming release - making it more clear when this happens and what could be done about it.

**Comment by jamieh-sky** [comment]
Hi @jlindbergh,

Thanks for the quick response. I broke up the troublesome line and the Code Health score was generated, I also did some clicking around and found the CodeScene Log which should hopefully mean we can catch these issues earlier.

It would be a handy feature to have a more visible message on the UI when something like this happens. We will look forward to the next release! 

Thanks again
